src/test/java/mate/academy/security/jwt/JwtTokenProviderTest.java

```diff
@@ -0,0 +1,143 @@
+package mate.academy.security.jwt;
+
+import java.util.List;
+import javax.servlet.http.HttpServletRequest;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.test.util.ReflectionTestUtils;
+
+class JwtTokenProviderTest {
+    private static UserDetailsService userDetailsService;
+    private static JwtTokenProvider jwtTokenProvider;
+    private static final String TOKEN = "eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJib2IiLCJyb2xlcyI6WyJV"
+            + "U0VSIl0sImlhdCI6MTY1ODI2MzcyNSwiZXhwIjoxNjU4MjY3MzI1fQ.9ujNBcbV9d5EE_C35CI1-3ykwmAr"
+            + "sq0pIAwxzAOBpdI";
+
+    @BeforeAll
+    public static void setUp() {
+        userDetailsService = Mockito.mock(UserDetailsService.class);
+        jwtTokenProvider = new JwtTokenProvider(userDetailsService);
+        ReflectionTestUtils.setField(jwtTokenProvider, "secretKey", "secret");
+        ReflectionTestUtils.setField(jwtTokenProvider, "validityInMilliseconds", 3600000);
+    }
+
+    @Test
+    public void createToken_ok() {
+        String login = "bob";
+        List<String> roles = List.of("USER");
+        String actual = jwtTokenProvider.createToken(login, roles);
+        Assertions.assertNotNull(actual);
+        Assertions.assertEquals(3, actual.split("\\.").length);

```

Comment:
```suggestion
        Assertions.assertEquals(3, actual.split("\\.").length);
```

you don't need to test logic that wasn't written by you, (the same for another method)